builtin/providers/aws/resource_aws_elb.go

```diff
@@ -85,7 +85,7 @@ func resourceAwsElb() *schema.Resource {
 			"listener": &schema.Schema{
 				Type:     schema.TypeSet,
 				Required: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
The approach used in other resources, is to only add the `ForceNew` option if it is set to `true`, otherwise just delete it from the schema. So in this case it would be better to just delete the line instead of setting it to `false`.